src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1023,13 +1023,6 @@ public function RemoveMLTConnection($sFieldName, $sConnectedID = false)
         /** @var TCMSMLTField $oField */
         $oField = $this->oTableConf->GetField($sFieldName, $this->oTable);
 
-        $sTargetTable = $oField->GetConnectedTableName();
```

Comment:
Note: During development i stumbled several times on this method and these part has no functionality. It loads a record, sets it to $oTargetTable but is not used anywhere. This looks like an orphan from the workflow feature or something. So i removed it, so other people will not scratch their heads like i did.

Reply:
For earlier versions this could replace the $sConectedId (something to do with translations).